dsc/reports/create_batch.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+import datetime
+import logging
+from io import StringIO
+from typing import TYPE_CHECKING, ClassVar
+
+import boto3
+import pandas as pd
+from jinja2 import Environment, FileSystemLoader, Template, select_autoescape
+
+from dsc.config import Config
+from dsc.item_submission import ItemSubmission
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from dsc import workflows
+
+logger = logging.getLogger(__name__)
+CONFIG = Config()
+
+
+class CreateBatchReport:
+    """Report class for Workflow.create_batch method.
+
+    The email created by this report is structured as follows:
+
+    1. A message summarizing the number of successfully created item
+       submissions in DynamoDB.
+    2. A CSV file containing all item submissions for a given batch
+       that were written to DynamoDB.
+    """
+
+    fields: ClassVar[list[str]] = [
+        "batch_id",
+        "item_identifier",
+        "source_system_identifier",
+        "status",
+        "status_details",
+        "dspace_handle",
+        "ingest_date",
+    ]
```

Comment:
I don't expect _all_ attributes are needed for the CSV files created by the report classes.